aws-cpp-sdk-s3control-integration-tests/S3ControlTest.cpp

```diff
@@ -644,81 +622,14 @@ namespace
         getAccessPointRequest.SetAccountId("123456789012");
         getAccessPointOutcome = m_client.GetAccessPoint(getAccessPointRequest);
         ASSERT_FALSE(getAccessPointOutcome.IsSuccess());
-        ASSERT_EQ(S3ControlErrors::VALIDATION, getAccessPointOutcome.GetError().GetErrorType());
+        EXPECT_EQ(CoreErrors::ENDPOINT_RESOLUTION_FAILURE, (CoreErrors) getAccessPointOutcome.GetError().GetErrorType());
 
         // Missing account ID
         getAccessPointRequest.SetName(accessPointName);
         getAccessPointRequest.SetAccountId("");
         getAccessPointOutcome = m_client.GetAccessPoint(getAccessPointRequest);
         ASSERT_FALSE(getAccessPointOutcome.IsSuccess());
-        ASSERT_EQ(S3ControlErrors::MISSING_PARAMETER, getAccessPointOutcome.GetError().GetErrorType());
-
-        // Missing outpost ID
-        ss.str("");
-        ss << "arn:aws:s3-outposts:" << Aws::Region::US_WEST_2 << ":" << m_accountId << ":outpost:" << "" << ":accesspoint:" << accessPointName;
```

Comment:
ARN parsing is deprecated
The test has been forgotten to be added to a previous commit